### PR TITLE
Creation of TEAbumper lambda #328

### PR DESCRIPTION
**Pull request by benbart**
Also updated the rain-api-core

### CODE REVIEW

File: cloudformation/thin-egress-app.yaml [hunk 1]
```diff
@@ -947,3 +947,59 @@ Resources:
...
+  BumperLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket: !Ref LambdaCodeS3Bucket
+        S3Key: !Ref LambdaCodeS3Key
+      Role: !GetAtt BumperLambdaIamRole.Arn
+      FunctionName: !Sub "${AWS::StackName}-BumperLambda"
+      VpcConfig:
+        !If
```
Comment (bbuechler) [inline cloudformation/thin-egress-app.yaml:959]: I honestly have no idea... If the function is in the VPC, can you call it on the cmd line from outside the vpc?

File: lambda/tea_bumper.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    log.debug(f"envvar for {TEA_LAMBDA_NAME}: {egress_env['Variables']}")
+    response = client.update_function_configuration(
+        FunctionName=TEA_LAMBDA_NAME,
+        Environment=egress_env
+    )
+    log.debug(response)
+
+
+def version():
```
Comment (bbuechler) [inline lambda/tea_bumper.py:30]: Is this to make logging work?

File: cloudformation/thin-egress-app.yaml (hunk 1 above)
Comment (benbart) [reply cloudformation/thin-egress-app.yaml:959]: I just copied it from the egress lambda. My understanding of networking is thin enough that I didn't want to mess with something I didn't understand. If I should remove it, I'd be happy to.

File: lambda/tea_bumper.py (hunk 2 above)
Comment (benbart) [reply lambda/tea_bumper.py:30]: It's to provide a hardcoded documentation of the version in file

**Pull request merged**